qgis-samgeo-plugin/samgeo_plugin.py

```
@@ -247,18 +297,150 @@ def unload(self):
+                self._deps_dock = None
+
+            # Now show the main plugin dock
+            if self.dock_widget is None:
+                self.dock_widget = self.create_dock_widget()
+                self.iface.addDockWidget(
+                    Qt.DockWidgetArea.RightDockWidgetArea, self.dock_widget
+                )
+            else:
+                self.dock_widget.show()
+
+    def _on_cancel_install(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_on_cancel_install`

Add return type annotation: `None`

(ANN202)